src/main/java/uk/gov/companieshouse/accounts/filing/service/file/validation/AccountsValidationServiceImpl.java

```diff
@@ -109,4 +117,16 @@ private RuntimeException validationStatusResultThrowableExceptions(final String
             throw new ResponseException(message);
         }
     }
+
+    // Method to get the filing type of the given packageType
+    private static String getAccountsFilingType(PackageTypeApi key) {
+        EnumMap<PackageTypeApi, String> accountsFilingTypeMap = new EnumMap<>(PackageTypeApi.class);
+        accountsFilingTypeMap.put(PackageTypeApi.UKSEF, "4");
+        accountsFilingTypeMap.put(PackageTypeApi.GROUP_PACKAGE_401, "4");
+        accountsFilingTypeMap.put(PackageTypeApi.OVERSEAS, "4");
+        accountsFilingTypeMap.put(PackageTypeApi.AUDIT_EXEMPT_SUBSIDIARY, "14");
+        accountsFilingTypeMap.put(PackageTypeApi.FILING_EXEMPT_SUBSIDIARY, "15");
```

Comment:
This will still recreate the map on every request. I was thinking like this: 
```suggestion
    private static final Map<PackageTypeApi, String> accountsFilingTypeMap = Map.of(
        PackageTypeApi.UKSEF, "4",
        PackageTypeApi.GROUP_PACKAGE_401, "4",
        ...
        );
```
Or you could create a function that does this so it's an Enum map but call the function on object load instead of every time a request is made.

Reply:
Suggestion accepted.